aliassetup/aliassetup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo ""  >> /etc/bash_alias
+chmod x /etc/bash_alias
```

Comment:
`chmod +x ...` ? ;-)
And perhaps this script could check that `/etc/bash_alias` doesn't already exist, before doing anything?

And then if you did that, I guess you could include it directly into the top of `addalias.sh`, which would allow you to remove the `aliassetup` block, making things easier to use for the user?
All IMHO of course.